### DISCUSSION TITLE
how to remove NEXTAUTH_URL warning when using vercel

### DISCUSSION THREAD

**Discussion by ldenblyd** [nextauthjs/next-auth#4331] [Help]
### Question 💬

hello i have these warning spamming my console

`[next-auth][warn][NEXTAUTH_URL]
https://next-auth.js.org/warnings#nextauth_url`

i dont need to set it because i use vercel 

there is a solution to remove this warning ?

thanks.

**Comment by EvHaus** [comment]
I think the answer you're looking for is here: https://next-auth.js.org/warnings#nextauth_url

**Reply by ldenblyd** [reply]
yes i have already checked this doc.

but it's incomplete i thinks

i have try to add 
NEXTAUTH_URL=http://localhost:3000
NEXTAUTH_URL=localhost:3000
NEXTAUTH_URL=0.0.0.0:3000
NEXTAUTH_URL=http://localhost

none of them works.

**Reply by ldenblyd** [reply]
and can we put it in env.local ?

also dont know

**Reply by aiyer-commits** [reply]
@ldenblyd Do you have a VERCEL="1" variable in your .env? if so, remove that and this warning will go away

**Reply by BorisHel** [reply]
Set NEXTAUTH_URL=http://localhost:3000 in your .env and restart your server to reload the configuration. That should work.
